validation/promotion_test.go

```diff
@@ -7,18 +7,23 @@ import (
 	. "github.com/ethereum-optimism/superchain-registry/superchain"
 )
 
+// WARNING: this test must not run along side any other tests, because it mutates global objects.
+// It should be strictly isolated.
 func TestPromotion(t *testing.T) {
 	for _, chain := range OPChains {
 		chain := chain
 		t.Run(perChainTestName(chain), func(t *testing.T) {
 			t.Parallel()
 			if chain.StandardChainCandidate {
+				// do not allow any test exclusions
+				exclusions = nil
 				// promote the chain to standard
 				// by mutating the chainConfig
 				chain.StandardChainCandidate = false
 				chain.SuperchainLevel = Standard
 				now := uint64(time.Now().Unix())
 				chain.SuperchainTime = &now
```

Comment:
Might be good to add a function `chainConfig.promoteToStandard()` to encapsulate those config modification steps

Reply:
Yes, I think we can do this in the superchain module and read it both from validation and add-chain modules. Will work on that.

Reply:
[90f7bf0](https://github.com/ethereum-optimism/superchain-registry/pull/483/commits/90f7bf076b43ba089380cfa4c248701a45fbca58)